ext/zend_test/test_arginfo.h

```diff
@@ -295,6 +295,11 @@ static const zend_function_entry class_ZendTestNS2_ZendSubNS_Foo_methods[] = {
 	ZEND_FE_END
 };
 
+static void register_test_symbols(int module_number)
+{
+	REGISTER_LONG_CONSTANT("ZEND_TEST_DEPRECATED", 42, CONST_CS | CONST_PERSISTENT | CONST_DEPRECATED);
```

Comment:
This adds `CONST_CS` which is a NOOP. Should `CONST_CS` be removed from the stub generator?

Reply:
Yep. I've already added it to my to-do list to remove const_cs, and will implement it as soon as all constants are migrated.